src/ansys/mapdl/core/_commands/database/set_up.py

```diff
@@ -0,0 +1,467 @@
+        information, see `Advanced Radiosity Options
+        <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en/ans_the/Hlp_G_THEadvrad.html#themultrsymm>`_
+
+        This command is valid in any processor.
+        """
+        command = f"SAVE,{fname},{ext},,{slab}"
+        return self.run(command, **kwargs)
+
+    def stitle(self, nline: str = "", title: str = "", **kwargs):
+        r"""Defines subtitles.
+
+        Mechanical APDL Command: `/STITLE <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_STITLE.html>`_
+
+        Parameters
+        ----------
+        nline : str
+            Subtitle line number (1 to 4). Defaults to 1.
+
+        title : str
+            Input up to 70 alphanumeric characters. Parameter substitution may be forced within the title by
+            enclosing the parameter name or parametric expression within percent (%) signs. If ``Title`` is
+            blank, this subtitle is deleted.
+
+        Notes
+        -----
+
+        .. _s-STITLE_notes:
+
+        Up to four subtitles are displayed in the output along with the main title ( :ref:`title` ).
+
+        Subtitles do not appear in GUI windows or in plot displays.
+
+        The first subtitle is also written to various Mechanical APDL files along with the main title.
+
+        Previous subtitles can be overwritten or deleted.
+
+        Issue :ref:`slashstatus` to display titles.
+
+        This command is valid in any processor.
+        """
+        command = f"/STITLE,{nline},{title}"
+        return self.run(command, **kwargs)
+
+    def stat(self, **kwargs):
+        r"""Displays the status of database settings.
+
+        Mechanical APDL Command: `STAT <https://ansyshelp.ansys.com/Views/Secured/corp/v232/en//ans_cmd/Hlp_C_STAT.html>`_
+
+        Notes
+        -----
+
+        .. _STAT_notes:
+
+        :ref:`stat` is a command generated by the GUI and will appear in the log file ( :file:`Jobname.LOG`
+        ) if status is requested for some items under Utility Menu> List> Status. Generally, :ref:`stat`
+        will be preceded by one of the commands listed below, which specifies the particular topic that
+        status was requested for.
+
+        If entered directly into the program, the :ref:`stat` command should be immediately preceded by the
+        desired topic command listed below. In processors other than those listed below (for example,
+        AUX12), no topic command should proceed :ref:`stat`.
+
+        This command is valid in any processor.
+
+        PREP7 topic commands (and their corresponding topics) are:
+
+        InformalTables need to be added.
+        SOLUTION topic commands (and their corresponding topics) are:
+
+        InformalTables need to be added.
+        POST1 topic commands (and their corresponding topics) are:
+
+        InformalTables need to be added.
+        POST26 topic commands (and their corresponding topics) are:
+
+        InformalTables need to be added.
```

Comment:
I hope you took note of this TODOs

Reply:
An [issue](https://github.com/ansys/pyconverter-xml2py/issues/383) is already opened in PyConverter-XML2PY